targets/TARGET_NUVOTON/mbed_rtx.h

```diff
@@ -19,7 +19,7 @@
 
 #include <stdint.h>
 
-#if defined(TARGET_NUMAKER_PFM_NUC472)
+#if defined(TARGET_NUVOTON)
```

Comment:
I have seen this change in another PR to add another nuvoton target - I dont see any dependency, so this one might state it and will be rebased ?